static/app/views/issueList/filters.tsx

```diff
@@ -68,4 +73,4 @@ const SearchContainer = styled('div')`
   }
 `;
 
-export default IssueListFilters;
+export default withRouter(IssueListFilters);
```

Comment:
Can't `useRouter` right?

Reply:
Not until we fully convert `overview.spec.jsx` unfortunately